docs/src/layout/NavigationMenu.jsx

```diff
@@ -43,6 +43,10 @@ const NavigationMenu = ({ classes }) => (
     <Link to="/guides/formats" className={classes.navLink}>
       <ListItem button> Global format customization </ListItem>
     </Link>
+
+    <Link to="/guides/persian" className={classes.navLink}>
```

Comment:
I suppose it should be under `Localization` section :)

Reply:
I thought that was for using different libraries. That was my first thought anyway, I'll move it 👍